### PR TITLE
Add easter egg for April 1st and October 31st

### PR DESCRIPTION
**Pull request by ccuser44**
Add easter egg for April 1st and October 31st

We do a little trolling

### CODE REVIEW

**Comment by Dimenpsyonal** [comment]
what is lifeoftheparty anyway

**Comment by ccuser44** [comment]
> what is lifeoftheparty anyway

Mystery

**Comment by Dimenpsyonal** [comment]
For all I know this is adding the env virus

**Comment by ccuser44** [comment]
> For all I know this is adding the env virus

Run the command `?ILOVEYOU`

**Comment by Dimenpsyonal** [comment]
well having this effect might be disruptive to games

**Comment by ccuser44** [comment]
> well having this effect might be disruptive to games

Just a prank bro ¯\_(ツ)_/¯

File: MainModule/Client/UI/Default/HelpButton.lua [hunk 1]
```diff
@@ -12,6 +12,12 @@ return function(data, env)
 	end
 
 	local UI = client.UI
+	local Functions = client.Functions
+
+	if Functions.CheckDate(1, 4) or Functions.Checkdate(31, 10) then
+		UI.MakeGui("Effect", {Mode = "lifeoftheparty";})
+		return
```
Comment (SuperCater) [inline MainModule/Client/UI/Default/HelpButton.lua:19]: Remove the return statement.

We shouldn't fully disable the help button if this was to be added

**Review by SuperCater — CHANGES_REQUESTED** [review]

**Comment by Dimenpsyonal** [comment]
After a review of the PR it doesn't appear to stop
image

**Comment by ccuser44** [comment]
> After a review of the PR it doesn't appear to stop
> ￼

:clearadonisguis me

**Comment by ccuser44** [comment]
Or reset

**Pull request closed without merging**